Description: include <asm/ptrace.h> instead of <asm/ptrace-abi.h>, which
 doesn't exist on armhf/arm64. On systems where it exists, it is being included
 by <asm/ptrace.h> anyway. This tweaks the change introduced at
 https://chromium.googlesource.com/chromium/src/+/8a33234fa1f1616545a9f0b3cc5b2fdf74b50c84.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc
+++ b/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc
@@ -36,9 +36,9 @@
 #include <sys/ptrace.h>
 #if !defined(PTRACE_GET_THREAD_AREA) && defined(OS_LINUX) && \
     !defined(OS_CHROMEOS)
-// Also include asm/ptrace-abi.h since ptrace.h in older libc (for instance
+// Also include asm/ptrace.h since ptrace.h in older libc (for instance
 // the one in Ubuntu 16.04 LTS) is missing PTRACE_GET_THREAD_AREA.
-#include <asm/ptrace-abi.h>
+#include <asm/ptrace.h>
 #endif  // !PTRACE_GET_THREAD_AREA && OS_LINUX && !OS_CHROMEOS
 #endif  // !OS_NACL_NONSFI
 
